PdCore/src/main/java/org/puredata/android/service/PdService.java

```diff
@@ -241,8 +241,14 @@ private Notification makeNotification(Intent intent, int icon, String title, Str
 				notificationManager.createNotificationChannel(channel);
 			}
 		}
-
-		PendingIntent pi = PendingIntent.getActivity(getApplicationContext(), 0, intent, 0);
+		
+            	PendingIntent pi = null;
+            	if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
I am not sure about setting `FLAG_UPDATE_CURRENT` here. On the one hand, this will just replace an existing notification so it will not show multiple notifications, if `startAudio()` is called multiple times. On the other hand, this should have been prevented in `startAudio()`.